postgraphiql/src/components/PostGraphiQL.js

```diff
@@ -93,6 +99,21 @@ class PostGraphiQL extends React.PureComponent {
       POSTGRAPHILE_CONFIG.enhanceGraphiql && POSTGRAPHILE_CONFIG.subscriptions ? 'pending' : null,
   };
 
+  constructor(params) {
+    super(params);
+
+    const saveHeadersText = JSON.parse(
+      this._storage.get(STORAGE_KEYS.SAVE_HEADERS_TEXT) || 'false',
+    );
+    if (saveHeadersText) {
+      this.state.saveHeadersText = saveHeadersText;
+      const headersText = this._storage.get(STORAGE_KEYS.HEADERS_TEXT);
+      if (headersText) {
+        this.state.headersText = headersText;
+      }
+    }
+  }
```

Comment:
Please move this code to the `state =` property initialiser above, and simplify it, e.g.

```js
    saveHeadersText: this._storage.get(STORAGE_KEYS.SAVE_HEADERS_TEXT) === 'true' ? true : false,
    headersText: this._storage.get(STORAGE_KEYS.HEADERS_TEXT) || "",
```

(Note headersText doesn't need to respect saveHeadersText here - if it's in local storage then we should display it anyway. The code later should ensure that it's not stored unless it's checked.)

Reply:
Okay. You sure you want an empty string instead of the default `'{\n"Authorization": null\n}\n'`up until now?

Reply:
Use the existing default :) The above comment is just an approximation.